dashboards-observability/public/components/explorer/home_table/history_table.tsx

```diff
@@ -88,14 +144,13 @@ export function Table(options: TableData) {
   return (
     <div>
       <EuiSpacer size="xl" />
-      <EuiBasicTable
-        items={queries.slice(pageIndex * pageSize, pageIndex * pageSize + pageSize)}
+      <EuiInMemoryTable
```

Comment:
using InMemoryTable requires storing all items in memory instead of a constant 10 items (which is `pageSize`, although I see it was not implemented previously either). will there be a performance issue when number of queries is large?

Reply:
I'm not sure about that part, i'll have a talk with Eric regarding this